priv/repo/migrations/20200518135714_login_log.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Auth.Repo.Migrations.LoginLog do
+  use Ecto.Migration
+
+  def change do
+    create table(:user_agents) do
+      add :name, :string
+    end
+
+    create table(:login_logs) do
+      add :email, :binary
```

Comment:
We don't just want to store an _unencrypted_ Binary.
We want to use `Fields.EmailEncrypted` to protect privacy in the case of a DB compromise.

Reply:
If we add `Fields.EmailEncrypted` we won't be able to create the log when a user is sending a wrong email format

Reply:
@SimonLab good point. I'd say 99.9% of the time the Browser will detect an invalid email address. But yeah, let's use `Fields.Encrypted`.